Geometry/RPCGeometryBuilder/src/RPCGeometryBuilder.h

```diff
@@ -32,26 +31,25 @@ class RPCDetId;
 class RPCRoll;
 class MuonGeometryConstants;
 
-class RPCGeometryBuilderFromDDD {
+class RPCGeometryBuilder {
 public:
-  RPCGeometryBuilderFromDDD(bool comp11);
+  RPCGeometryBuilder(bool comp11);
 
-  ~RPCGeometryBuilderFromDDD();
+  ~RPCGeometryBuilder();
```

Comment:
in C++11 and above you do not need to write an empty destructor. Please, remove it.

Reply:
done